CONTRIBUTING.md

```diff
@@ -47,137 +105,154 @@ the [Software Carpentry introduction to git][link_swc_intro].
+    Make sure to [keep your fork up to date][link_updateupstreamwiki] with the upstream repository.<br />  
+    For example, to update your master branch on your local cloned repository:  
+      ```Shell
+      git fetch upstream
+      git checkout master
+      git merge upstream/master
+      ```
+
+1. **Create a [new branch][link_branches] to develop and maintain the proposed code changes.**<br />
+  For example:
+    ```Shell
+    git fetch upstream  # Always start with an updated upstream
+    git checkout -b fix/bug-1222 upstream/master
+    ```
+    Please consider using appropriate branch names as those listed below, and mind that some of them
+    are special (e.g., `doc/` and `docs/`):
+      * `fix/<some-identifier>`: for bugfixes
+      * `enh/<feature-name>`: for new features
+      * `doc/<some-identifier>`: for documentation improvements.
+        You should name all your documentation branches with the prefix `doc/` or `docs/`
+        as that will preempt triggering the full battery of continuous integration tests.
+
+1. **Make the changes you've discussed, following the [fMRIPrep coding style guide](#fMRIPrep-coding-style-guide).**<br />
+  Try to keep the changes focused: it is generally easy to review changes that address one feature or bug at a time.
+  It can also be helpful to test your changes locally,
+  using a [fMRIPrep development environment][link_devel].
+  Once you are satisfied with your local changes, [add/commit/push them][link_add_commit_push]
+  to the branch on your forked repository.
+
+1. **Submit a [pull request][link_pullrequest].**<br />
+   A member of the development team will review your changes to confirm
+   that they can be merged into the main code base.<br />
+   Pull request titles should begin with a descriptive prefix
+   (for example, `ENH: Support for SB-reference in multi-band datasets`):  
+     * `ENH`: enhancements or new features ([example][enh_ex])
+     * `FIX`: bug fixes ([example][fix_ex])
+     * `TST`: new or updated tests ([example][tst_ex])
+     * `DOC`: new or updated documentation ([example][doc_ex])
+     * `STY`: style changes ([example][sty_ex])
+     * `REF`: refactoring existing code ([example][ref_ex])
+     * `CI`: updates to continous integration infrastructure ([example][ci_ex])
+     * `MAINT`: general maintenance ([example][maint_ex])
+     * For works-in-progress, add the `WIP` tag in addition to the descriptive prefix.
+       Pull-requests tagged with `WIP:` will not be merged until the tag is removed.
+
+1. **Have your PR reviewed by the developers team, and update your changes accordingly in your branch.**<br />
+   The reviewers will take special care in assisting you address their comments, as well as dealing with conflicts
+   and other tricky situations that could emerge from distributed development.
+   And if you don't make the requested changes, we might ask
+   [@bedevere-bot](https://github.com/search?q=commenter%3Abedevere-bot+soft+cushions) 
```

Comment:
AFAICT this bot links to a Youtube video that a lot of people can't see due to copyright restrictions. I'm not sure it's worth the joke.

Reply:
I'm fine either way. Without the video we might just remove the joke. I added it because to signify that fMRIPrep is done in Python (and hence the Monty Python theme).

Reply:
Any more voices? If none, I'll remove the joke :+1:

Reply:
I can't see it :( I like the joke, but +1 on removing just for those of us who would miss the video !

Reply:
I'm actually curious who can see it... Maybe just Brits.